### PR TITLE
Add foobar2000

### PR DESCRIPTION
**Pull request by tlm-2501**
A music player.

### CODE REVIEW

File: just-install.json [hunk 1]
```diff
@@ -730,6 +730,19 @@
       },
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
```
Comment (geek1011) [inline just-install.json]: foobar2000 uses NSIS for it's installer.

```suggestion
        "kind": "nsis",
```

File: just-install.json [hunk 2]
```diff
@@ -730,6 +730,19 @@
...
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/S"
+          ]
+        }
```
Comment (geek1011) [inline just-install.json]:
```suggestion
```

File: just-install.json [hunk 3]
```diff
@@ -730,6 +730,19 @@
...
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/S"
+          ]
```
Comment (geek1011) [inline just-install.json]:
```suggestion
```

File: just-install.json [hunk 4]
```diff
@@ -730,6 +730,19 @@
...
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
+            "/S"
```
Comment (geek1011) [inline just-install.json]:
```suggestion
```

File: just-install.json [hunk 5]
```diff
@@ -730,6 +730,19 @@
       },
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
+        "options": {
+          "arguments": [
+            "{{.installer}}",
```
Comment (geek1011) [inline just-install.json]:
```suggestion
```

File: just-install.json [hunk 6]
```diff
@@ -730,6 +730,19 @@
       },
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
+        "options": {
+          "arguments": [
```
Comment (geek1011) [inline just-install.json]:
```suggestion
```

File: just-install.json [hunk 7]
```diff
@@ -730,6 +730,19 @@
       },
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
+        "options": {
```
Comment (geek1011) [inline just-install.json]:
```suggestion
```

File: just-install.json [hunk 8]
```diff
@@ -730,6 +730,19 @@
       },
       "version": "latest"
     },
+    "foobar2000": {
+      "installer": {
+        "kind": "custom",
+        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe",
```
Comment (geek1011) [inline just-install.json]:
```suggestion
        "x86": "https://www.foobar2000.org/files/2c3b9ba9af4ddfa2b720112a7c498c5a/foobar2000_v1.4.3.exe"
```

**Comment by tlm-2501** [comment]
Applied the relevant change. Thank you.

**Comment by geek1011** [comment]
No problem! Also, note that you can batch the changes from the files tab.

**Pull request merged**